jans-cedarling/bindings/cedarling_python/example_files/policy-store.json

```diff
@@ -2,15 +2,15 @@
     "cedar_version": "v2.4.7",
     "cedar_policies": {
         "840da5d85403f35ea76519ed1a18a33989f855bf1cf8": {
-            "description": "simple policy example for principal workload",
+            "description": "simple policy example for pricipal workload",
```

Comment:
```suggestion
            "description": "simple policy example for principal workload",
```

Reply:
Fixed in [commit](https://github.com/JanssenProject/jans/pull/10016/commits/ca2e503d76279f88ef704b814b4e4d16ab621697)